### PR TITLE
Fixes color flash for add funds button

### LINKED ISSUE #4066: On browser startup, Preferences > Payments briefly shows off/disabled view
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**

When the last active tab was Preferences > Payments and you restart the browser, the Payments page briefly in the Payments off state even with Payments on.

Mentioned in https://github.com/brave/browser-laptop/issues/3781#issuecomment-247658679

**Expected behavior:**

Payments page should always accurately reflect Payments on/off status.
- Platform (Win7, 8, 10? macOS? Linux distro?):
  Mac OS
- Brave Version:
  0.12.1
- Steps to reproduce:
  1. Enable Payments
  2. Open 10 tabs of stuff
  3. Open Preferences > Payments
  4. Restart browser
  5. Browser reloads Payments page in the off state

### LINKED ISSUE #11387: Ledger is shown as enabled for a split second
### Description

When you are enabling brave payment and you don't have existing wallet, we display add founds button orange for a split second, before transforming it into disabled state until wallet is creating.
### Steps to Reproduce

  1. clean profile
  2. enable brave payments
  3.

**Actual result:**

button is orange for a split second
**Expected result:**
button should be greyed out all the time until the wallet is created
**Reproduces how often:** [What percentage of the time does it reproduce?]

### Brave Version

**about:brave info:**

**Reproducible on current live release:**

### PR DESCRIPTION
**Pull request by NejcZdovc**
Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Resolves #11387 
Resolves #4066

Auditors:

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/renderer/components/preferences/payment/enabledContent.js [hunk 1]
```diff
@@ -170,6 +170,7 @@ class EnabledContent extends ImmutableComponent {
 
   render () {
     const ledgerData = this.props.ledgerData
+    console.log(ledgerData.toJS())
```
Comment (cezaraugusto) [inline app/renderer/components/preferences/payment/enabledContent.js]: oops

**Comment by cezaraugusto** [comment]
I think this definitely solves #11397 too

**Comment by cezaraugusto** [comment]
Looks good but setting as blocked until #11397

File: app/renderer/components/preferences/payment/enabledContent.js (hunk 1 above)
Comment (NejcZdovc) [reply app/renderer/components/preferences/payment/enabledContent.js]: crap :smile:

**Comment by cezaraugusto** [comment]
master 2cf81bd
0.20.x 015613087863137cf832dd7fe53d6fb874ea481c
0.19.x a3fe13c769cb1b51c96e78742b8620bd120112c6

**Pull request merged**